### PR TITLE
DDF-4071 Removed embedded jetty-servlets

### PR DESCRIPTION
**Pull request by tyler30clemens**
#### What does this PR do?
There is no need to embed these jars since the artifact is a bundle that is already running by default. Several other bundles are already importing the packages exported by the jetty-servlets bundle.

#### Who is reviewing it? 
@jhunzik 
@ethantmanns 
@paouelle 

#### Ask 2 committers to review/merge the PR and tag them here.
@tbatie 
@figliold 

#### How should this be tested?
1. Build and install DDF through Admin UI.
2. Open "Inspect" to "Network" tab.
3. Navigate to search-redirect by hitting https://localhost:8993/search (which will redirect you)
   - check for "Content Encoding: gzip" to make sure and verify GzipFilter is occurring. 
4. Navigate to https://localhost:8993/ogc/gml/2.1.2/feature.xsd 
   - check for "Content Encoding: gzip" to make sure and verify GzipFilter is occurring.

#### What are the relevant tickets?
[DDF-4071](https://codice.atlassian.net/browse/DDF-4071)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by paouelle** [comment]
❗️ How will that impact the security manager?
If permissions were required by this code, they would have been added to the bundle where it was embedded. If we are no longer embedding than shouldn't we be cleaning up those permissions from the policy file?

**Comment by tyler30clemens** [comment]
The permissions were never explicitly added in the original PR (https://github.com/codice/ddf/pull/2159/files). If they were added after, they seem to have already been removed from the policy file with regards to these specific changes.

File: catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml [hunk 1]
```diff
@@ -42,9 +33,6 @@
                         <Web-ContextPath>/ogc</Web-ContextPath>
                         <_wab>src/main/resources</_wab>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>
```
Comment (figliold) [inline catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml]: :question: If you don't embed jetty-servlets, wouldn't you need to import the packages exported by the jetty-servlets bundle?

File: catalog/ui/search-ui/search-redirect/pom.xml [hunk 2]
```diff
@@ -64,9 +58,6 @@
                         <!--<Web-ContextPath>/search</Web-ContextPath>-->
                         <!--<_wab>src/main/webapp</_wab>-->
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>
```
Comment (figliold) [inline catalog/ui/search-ui/search-redirect/pom.xml]: ❓ If you don't embed jetty-servlets, wouldn't you need to import the packages exported by the jetty-servlets bundle?

File: catalog/ui/search-ui/search-redirect/pom.xml [hunk 3]
```diff
@@ -27,12 +27,6 @@
             <groupId>org.osgi</groupId>
             <artifactId>org.osgi.core</artifactId>
         </dependency>
-        <dependency>
```
Comment (figliold) [inline catalog/ui/search-ui/search-redirect/pom.xml]: ❓ If we are going to import packages from jetty-servlets bundle, wouldn't we still want jetty-servlets as a dependency?

File: catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml [hunk 4]
```diff
@@ -23,15 +23,6 @@
     <name>DDF :: Spatial :: WFS :: 1.0.0 :: Schema WebApp</name>
     <packaging>bundle</packaging>
 
-    <dependencies>
-        <dependency>
```
Comment (figliold) [inline catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml]: ❓ If we are going to import packages from jetty-servlets bundle, wouldn't we still want jetty-servlets as a dependency?

File: catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml (hunk 1 above)
Comment (tyler30clemens) [reply catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml]: The artifact is already running by default and several other bundles are already importing the packages.

File: catalog/ui/search-ui/search-redirect/pom.xml (hunk 3 above)
Comment (tyler30clemens) [reply catalog/ui/search-ui/search-redirect/pom.xml]: in the original PR (https://github.com/codice/ddf/pull/2159/files), it seems as though the dependency was only added to help with GzipFilter functionality. After these changes, it was still working. Therefore, it seems unnecessary at this point.

File: catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml (hunk 4 above)
Comment (tyler30clemens) [reply catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml]: Before these changes, I was unable to navigate to https://localhost:8993/ogc/gml/2.1.2/feature.xsd
and make sure it shows the feature.xsd file. I'm not sure if this is important, but this was what I used to test the changes that were added in the original PR.

**Comment by tyler30clemens** [comment]
build now

**Comment by tyler30clemens** [comment]
build now

**Comment by tyler30clemens** [comment]
build now

**Comment by tyler30clemens** [comment]
build now

**Comment by tyler30clemens** [comment]
build now

**Comment by tyler30clemens** [comment]
build now (flaky yarn installation)

**Comment by tyler30clemens** [comment]
build now

**Comment by tyler30clemens** [comment]
build now

**Comment by jhunzik** [comment]
Hero'd successfully

**Pull request merged**